tests/saitests/sai_qos_tests.py

```diff
@@ -1983,19 +1989,24 @@ def runTest(self):
 
         # send packets
         try:
+            # Read initial counters to set the baseline
+            q_wm_res_bases, pg_shared_wm_res_bases, pg_headroom_wm_res_bases = sai_thrift_read_port_watermarks(self.client, port_list[dst_port_id])
```

Comment:
The same comments above apply for the queue watermark as well

Reply:
Updated.